packages/@aws-cdk/aws-stepfunctions-tasks/lib/athena/start-query-execution.ts

```diff
@@ -87,7 +87,7 @@ export class AthenaStartQueryExecution extends sfn.TaskStateBase {
           }),
 
         ],
-        actions: ['athena:getDataCatalog', 'athena:startQueryExecution'],
+        actions: ['athena:getDataCatalog', 'athena:startQueryExecution', 'athena:getQueryExecution'],
```

Comment:
can you add the rationale for this addition into the commit body. I didn't see it in there

Reply:
added a link of this documentation: https://docs.aws.amazon.com/step-functions/latest/dg/athena-iam.html